CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+### 0.7.1
+
+* Add `runStmt` for executing statements in the `IO` monad and binding new names.
+
```

Comment:
Ok! Sorry about this, I just wanted to reduce your burden of having to update numbers everywhere, but was apparently overly eager.